pbxproj/pbxcli/pbxproj_file.py

```diff
@@ -52,14 +53,24 @@ def _add(project, args):
     else:
         header_scope = args['--header-scope']
 
+    parent_group=None
+    if args['--parent']:
+        parent_group = project.get_or_create_group(args['--parent'])
+
     options = FileOptions(create_build_files=not args['--no-create-build-files'],
                           weak=args['--weak'],
                           ignore_unknown_type=args['--ignore-unknown-types'],
                           embed_framework=not args['--no-embed'],
                           code_sign_on_copy=args['--sign-on-copy'],
                           header_scope=header_scope.title())
-    build_files = project.add_file(args['<path>'], tree=args['--tree'], force=False, target_name=args['--target'],
-                                   file_options=options)
+    build_files = None
+    if parent_group:
+        build_files = project.add_file(args['<path>'], tree=args['--tree'], force=False, target_name=args['--target'],
+                                    parent=parent_group, file_options=options)
+    else:
+        build_files = project.add_file(args['<path>'], tree=args['--tree'], force=False, target_name=args['--target'],
+                                    file_options=options)
+
```

Comment:
the _if-else_ seems unnecessary, `parent` defaults to `None` when not passed, if `parent_group` is `None` it will be equivalent

Reply:
Agreed. I applied the code changes as you suggested.